apps/src/gamelab/GameLabP5.js

```diff
@@ -427,6 +428,22 @@ GameLabP5.prototype.init = function (options) {
     array.displace = _groupCollideGameLab.bind(array, 'displace');
     array.bounce = _groupCollideGameLab.bind(array, 'bounce');
 
+    /**
+     * Test each member of group against the target using the isTouching sprite
+     * method.  Return true if any touching occurred.
+     *
+     * @method isTouching
+     * @param {Object} target Group or Sprite
+     * @return {boolean} True if any touching occurred
+     */
+    array.isTouching = function (target) {
+      var didTouch = false;
+      for (var i = 0; i < this.size(); i++) {
```

Comment:
This is just copying the _groupCollide() that you added to `p5.play.js` - I'm happy to change it!